src/lucky/uploaded_file.cr

```diff
@@ -14,6 +18,11 @@ class Lucky::UploadedFile
       HTTP::FormData.parse_content_disposition(part.headers["Content-Disposition"]).last
   end
 
+  # Returns the path of the tempfile as a String
+  #
+  #```
+  #uploaded_file_object.path #=> String
+  #``` 
```

Comment:
I think the build is failing because there's an extra space at the end of this line. It's _very_ picky.